settings.gradle

```diff
@@ -126,8 +126,10 @@ include ":sdks:python:container:py3"
 include ":sdks:python:test-suites:dataflow"
 include ":sdks:python:test-suites:dataflow:py35"
 include ":sdks:python:test-suites:dataflow:py36"
+include ":sdks:python:test-suites:dataflow:py37"
 include ":sdks:python:test-suites:direct:py35"
 include ":sdks:python:test-suites:direct:py36"
+include ":sdks:python:test-suites:direct:py36"
```

Comment:
I believe a typo here is breaking python postcommits: should be `py37`?

Reply:
Example log: https://builds.apache.org/job/beam_PostCommit_Python3_Verify_PR/321/console

Reply:
Thank you, @udim. Sent https://github.com/apache/beam/pull/8601. For some reason the error didn't happen in postcommit run on this PR.

Reply:
Look like most recent test run on this PR did not include postcommits, I overlooked it.